taskvine/src/manager/vine_task.c

```diff
@@ -99,6 +100,8 @@ void vine_task_clean(struct vine_task *t)
 
 	free(t->output);
```

Comment:
rename all `t->output` and `t->output_length` to `t->std_output`? It gets confusing when tasks have other outputs.

Reply:
Actually @JinZhou5042 just made a similar change in a different pending PR, I would recommend against also changing that here, just to avoid further conflicts.

Reply:
Do we use `stdout`/`stdout_length` or `std_output`/`std_output_length`?

Reply:
Remains unchanged to avoid conflicts.